examples/lfric/eg17/README.md

```diff
@@ -0,0 +1,54 @@
+# PSyData Examples
+
+**Author:** J. Henrichs, Bureau of Meteorology
+
+This directory contains some runnable examples of LFRic. They are based
+on a subset of the LFRic infrastructure library, that has been slightly
+modified to make it easier to create stand-alone, non-MPI LFRic codes.
+The examples here show:
+- A [full_example](./full_example) of a stand-alone LFRic-based code. It
+  uses PSyclone for a very simple kernel.
+- [Extraction](./full_example_extract) of input- and output-parameter
+  of a kernel to NetCDF files.
+- A stand-alone example using [NetCDF](./full_example_netcdf) to read
+  in a mesh.
+
+Detailed instructions are in the various subdirectories.
+
+## Licence
```

Comment:
I'm not sure that the licence needs to be explicitly visible. How about hiding it with HTML comments? What do @rupertford and @arporter think?

Reply:
I am in favour of putting the license to the end, since there (imho) it is really annoying to first have to scroll to find the actual README. I don't mind about hiding or not, but if we hide it we then should also hide the ``## License`` section (though I am wondering, why hide it??). I'll get a consensus at the telco ;)

Reply:
We didn't really agree on anything. I've added HTML comments, and we can decide later how we handle READMEs.

Reply:
OK - I am confused ... and left it just the way it was.

Reply:
Thanks